Desktop/data/computedcolumnsmodel.cpp

```diff
@@ -175,6 +175,17 @@ void ComputedColumnsModel::revertToDefaultInvalidatedColumns()
 			DataSetPackage::pkg()->columnSetDefaultValues(col->name());
 }
 
+void ComputedColumnsModel::computeColumnRemoved(QString columnNameQ)
+{
+	std::string columnName	= columnNameQ.toStdString();
```

Comment:
columnname is not used, so this can be removed

Reply:
yeah well, as i mentioned somewhere, this signal/slot isnt currently used.
but i can easily see how this would be useful if jaspColumn gets tracked with proper dependencies